src/eastwood/linters/implicit_dependencies.clj

```diff
@@ -0,0 +1,31 @@
+(ns eastwood.linters.implicit-dependencies
+  (:require [eastwood.copieddeps.dep1.clojure.tools.analyzer.ast :as ast]
+            [eastwood.copieddeps.dep9.clojure.tools.namespace.parse :as ns-parse]
+            [eastwood.util :as util]))
+
+
+(defn var->ns-symbol [var]
+  (let [^clojure.lang.Namespace ns (-> var meta :ns)]
```

Comment:
Just curious, why not use `(namespace var)`. 
A good answer would be that "That doesn't work" :)

Reply:
Doesn't work = )
`clojure.core/namespace`'s docstring: "Returns the namespace String of a symbol or keyword, or nil if not present."

If you try it: `(namespace #'map)`, you get `ClassCastException clojure.lang.Var cannot be cast to clojure.lang.Named`